instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -595,7 +600,7 @@ def _instrumented_lambda_handler_call(  # noqa pylint: disable=too-many-branches
                         if lambda_event["Records"][0]["eventSource"] == "aws:sqs":
                             span.set_attribute(SpanAttributes.FAAS_TRIGGER, "pubsub")
                             span.set_attribute("messaging.message",
-                                            limit_string_size(lambda_event["Records"]))
+                                            limit_string_size(lambda_event["Records"][0].get("body")))
```

Comment:
I'm not sure how is this related to the PR, can you please elaborate what is implemented here 🙏🏼